### PR TITLE
Fix handlers path params

### PR DESCRIPTION
**Pull request by Jacob4649**
Misc fixes to resolve handler conflict and make all request codes constant.

### CODE REVIEW

**Review by BullDF — APPROVED** [review]

**Review by Enkhzul001 — APPROVED** [review]
looks good

**Review by yuanshu777 — APPROVED** [review]
Thanks for fixing the bug

File: backend/src/main/java/com/javahelp/backend/endpoint/HTTPTokenHandler.java [hunk 1]
```diff
@@ -31,7 +31,7 @@ public abstract class HTTPTokenHandler extends HTTPHandler implements ITokenAuth
     private String tokenString, userId;
 
     @Override
-    public APIGatewayResponse getResponse(JsonObject body, HttpMethod method, Map<String, String[]> headers, Map<String, String[]> parameters) {
+    public APIGatewayResponse getResponse(JsonObject body, HttpMethod method, Map<String, String[]> headers, Map<String, String[]> parameters, Map<String, String> pathParameters) {
```
Comment (Lam207) [inline backend/src/main/java/com/javahelp/backend/endpoint/HTTPTokenHandler.java:34]: Would this extra parameter be reflected in some other changes somewhere else?

**Review by Lam207 — APPROVED** [review]
Looks good to me

File: backend/src/main/java/com/javahelp/backend/endpoint/HTTPTokenHandler.java (hunk 1 above)
Comment (Jacob4649) [reply backend/src/main/java/com/javahelp/backend/endpoint/HTTPTokenHandler.java:34]: That should be everywhere

**Pull request merged**